index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio | M Nisar Ali</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Playfair Display' rel='stylesheet'>
+    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Poppins" />
+</head>
+
+<body>
+    <header>
+        <a class="nav" href="#">
+            <img src="./Images/ic_menu.svg" alt="Navigation menu">
+        </a>
+    </header>
+    <main>
+        <section class="headline">
+            <h1>Muhammad Nisar Ali</h1>
+            <div class="intro">
+                <p>
+                    Hello! I am a software developer! I can help you build a product, feature
+                    or website. Take a look of my works. If you like what you see and have a
+                    project you need coded, don't hesitate and contact me.
+                </p>
+            </div>
+            <a class="collab-button" href="#">
+                <button class="button" type="button">Start collaboration</button>
+            </a>
+        </section>
+        <div class="bottom">
+            <img class="bottom-img" src="./Images/image_geometry_2.svg" alt=" ">
+        </div>
```

Comment:
- [ ] I think it would be better if you use this image in a `background-image` property from your `style.css` file instead of an `img` tag as required for the project. I suggest you use an empty `div` with a class, then from that class, you can style it as its background image in the CSS file.

Reply:
Made changes as per requirement.